addons/website/static/tests/tours/background_colorpicker.js

```diff
@@ -0,0 +1,69 @@
+/** @odoo-module **/
+
+import wTourUtils from "@website/js/tours/tour_utils";
+
+function selectColorpickerSwitchPanel(type) {
+    return [
+        {
+            content: "Select text snippet",
+            trigger: "iframe #wrap .s_text_block",
+        },
+        {
+            content: "Click on background-color option",
+            trigger: ".o_we_so_color_palette[data-css-property='background-color']",
+        },
+        {
+            content: "Select type of colorpicker in switch panel",
+            trigger: `.o_we_colorpicker_switch_pane_btn[data-target="${type}"]`,
+        },    
+    ]
+}
+
+function checkBackgroundColorWithRGBA(red, green, blue) {
+    return [
+        {
+            content: "Check if the RGBA color matches the selected color",
+            trigger: ".o_rgba_div",
+            run: ({ tip_widget }) => {
+                const rgbaEl = tip_widget.$anchor[0];
+                const red_color = rgbaEl.getElementsByClassName("o_red_input")[0].value;
+                const green_color = rgbaEl.getElementsByClassName("o_green_input")[0].value;
+                const blue_color = rgbaEl.getElementsByClassName("o_blue_input")[0].value;
```

Comment:
@paru-odoo use querySeletor instead
```suggestion
                const red_color = rgbaEl.querySelector("o_red_input").value;
                const green_color = rgbaEl.querySelector("o_green_input").value;
                const blue_color = rgbaEl.querySelector("o_blue_input").value;
```